Data-Science/Kubeflow/MNIST-Digits-Recognition/mnist_katib_tf_kserve_example.ipynb

```diff
@@ -388,12 +396,23 @@
     "    return \" \".join(best_hps)\n",
     "\n",
     "# You should define the TFJob name, namespace, number of training steps, output of Katib and model volume tasks in the arguments.\n",
-    "def create_tfjob_task(tfjob_name, tfjob_namespace, training_steps, katib_op, model_volume_op):\n",
+    "def create_tfjob_task(tfjob_name, tfjob_namespace, training_steps, katib_op, model_volume_op, http_proxy, no_proxy):\n",
     "    import json\n",
     "    # Get parameters from the Katib Experiment.\n",
     "    # Parameters are in the format \"--tf-learning-rate=0.01 --tf-batch-size=100\"\n",
-    "    convert_katib_results_op = components.func_to_container_op(convert_katib_results)\n",
-    "    best_hp_op = convert_katib_results_op(katib_op.output)\n",
+    "    best_hp_op = convert_katib_results(katib_results = katib_op.output)\n",
+    "    # Setting the proxy and no_proxy to allow download of the required packages inside the component. Uncomment the below code, if proxies are specified above.\n",
```

Comment:
Please remove or update the comment. In case of update please notice that it is not related to `set_caching_options` that sits just below it.

Reply:
Updated the comment and moved the `set_caching_options` part before the comment.